src/test/spec/initial_dns_seedlist_discovery.rs

```diff
@@ -102,7 +102,7 @@ async fn run() {
                 client
                     .drop_and_create_user(
                         user,
-                        pwd,
+                        pwd.as_str(),
```

Comment:
Unfortunately, even though `&String` can coerce to `&str` and `&str` implements `Into<Option<&str>>`, the compiler doesn't accept `&String` as an `impl Into<Option<&str>>`. I propose a new meme format called "Rust World Problems" (a la "first world problems") to refer to situations like this.